packages/hopli/src/identity.rs

```diff
@@ -0,0 +1,80 @@
+use crate::key_pair::create_identity;
+use clap::Parser;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use crate::utils::{Cmd, HelperErrors};
+
+/// CLI arguments for `hopli identity`
+#[derive(Parser, Default, Debug)]
+pub struct IdentityArgs {
```

Comment:
The command name is misleading, should be more specific like `create-identities`.

Reply:
Introduced a flag `--action` where `create` or `read` can be passed,  so that action on identities are specified.